### DISCUSSION TITLE
Geophires - PERCENTAGE OF TOTAL HEAT MINED (%)

### DISCUSSION THREAD

**Discussion by tb831637** [NatLabRockies/GEOPHIRES-X#265] [Q&A]
I recently ran a scenario and saw that this value was nearly 400% at EOFL. What is the ramification of that? Does one need to style the project to stay below 100%.

**Comment by malcolm-dsider** [comment]
Yes. You can't mine more heat than is there! In fact, there is a practical limit to what you can mine - perhaps depending a bit on the technology you use, but my sense is that the practical limit is something like 25%. After that, your performance would likely decline more rapidly.

**Reply by tb831637** [reply]
This hydrothermal project had injector/producer pairs.

**Reply by malcolm-dsider** [reply]
Well, truth be told, the percent of heat mined is a comparison between the produced and the heat available... but the heat available is a static value calculated once, at the beginning of the process. If new heat is being brought into the system thru recirculation/convection/conduction from outside the bounds of the system that you described, then it is possible to mine more heat that just the 25% I quote above... but I still don't think you will be able to mine anything close to 100%, let alone 400%, because the heat replacement methods are imperfect and slow.

**Reply by tb831637** [reply]
Seems like the program outputs should reflect that as one approaches some limit then... I don't see that in the graphics for this scenario.....
